libs/utils/trace.py

```diff
@@ -231,8 +215,16 @@ def __parseTrace(self, path, window, normalize_time, trace_format):
         else:
             raise ValueError("Unknown trace format {}".format(trace_format))
 
+        # If using normalized time, we should use
+        # TRAPpy's `abs_window` instead of `window`
+        window_kw = {}
```

Comment:
Added a dict for that window argument. I like this version because it only modifies the argument we're interested in. Of course, there's little chance that the default value will be changed from (0, None), but I prefer to be able to leave default values alone.

Reply:
Oh yeah this is nice, I didn't think of using `**` for just the winow arg. :+1: